templates/test/ci/cluster-template-prow-ci-version-windows-containerd-2022.yaml

```diff
@@ -216,8 +216,8 @@ spec:
         marketplace:
           offer: capi
           publisher: cncf-upstream
-          sku: k8s-1dot18dot8-ubuntu-1804
-          version: 2020.08.17
+          sku: ubuntu-1804-gen1
+          version: latest
```

Comment:
`latest` here isn't going to be very predictable, unless I misunderstand. It will depend on the order of publishing, not on semantic version, so `latest` will give a recently published version but it may change from `v1.24.1` one day to `v1.22.10` the next.

Reply:
Yes, that's a good point. I guess the point is that we don't care about k8s version on the image since we're replacing it with a CI version of k8s, we just need a fresh image (built with latest image-builder code) that isn't going to be removed and break tests in the future. We had a similar conversation with @jsturtevant in #2224. My main point is that the test is already unpredictable because from one run to the next the value of "latest" k8s ci version may have changed and it's better to test with latest image-builder code that an old image that was built months ago.

Reply:
the conversation was here: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2373#pullrequestreview-1003054672